filament/src/Renderer.cpp

```diff
@@ -1204,8 +1249,8 @@ void FRenderer::initializeClearFlags() {
                           TargetBufferFlags::COLOR : TargetBufferFlags::NONE)
                          | TargetBufferFlags::DEPTH_AND_STENCIL;
 
-    mClearFlags = (mClearOptions.clear ? TargetBufferFlags::COLOR : TargetBufferFlags::NONE)
-            | TargetBufferFlags::DEPTH_AND_STENCIL;
+    mClearFlags = (mClearOptions.clear ? (TargetBufferFlags::COLOR| TargetBufferFlags::DEPTH_AND_STENCIL) : TargetBufferFlags::NONE)
+            ;
```

Comment:
How about introducing 2 bool flags to give more control to the client: `clearDepth`, `clearColor`? It would still fit our use case but would be more upstreamable, and wouldn't force everyone to clear the color and depth buffers together.